src/main/java/hudson/plugins/sidebar_link/LinkAction.java

```diff
@@ -59,9 +62,14 @@ public LinkAction(String urlName, String displayName, String iconFileName) throw
             throw new IllegalArgumentException(validationResult);
         }
 
+        if((iconFileName == null) || (iconFileName.equals(""))) {
```

Comment:
StringUtils.isBlank(iconFileName) like below

Reply:
Done: please review "minor changes due to review"